CHANGELOG.md

```diff
@@ -1,3 +1,212 @@
+# cuDF 0.16.0 (Date TBD)
```

Comment:
Should this TBD be updated in this PR?

Reply:
Yes, it typically gets updated right before merging.